apps/sample-next-auth-app/README.md

```diff
@@ -1,60 +1,60 @@
 # Xata + Next.js + Auth.js
 
-Sample app using Next-Auth for managing OAuth login with the Xata adapter.
+This is a simple dashboard built using Next 13 with Auth.js to manage OAuth & the official [Xata adapter](https://authjs.dev/reference/adapter/xata) to capture user storage, sessions, and more!
 
-> ⚠️ This example uses the **App Routing**, if using `/pages`, go for **Pages Routing**.
+> ⚠️ This example uses the Next 13 `app router`. If you're using `page routing` you can find an example [here](https://github.com/xataio/examples/tree/main/apps/sample-next-auth-pages).
 
 ## Features ⚡️
 
 - React Server Components
-- Next-Auth API Routes.
-- GitHub as OAuth Provider with Xata official adapter.
+- Auth.js Route Handlers
+- GitHub as OAuth Provider
+- User sessions, storage, and more in your database using the Xata adapter
 
 ### Cloning the Project
 
+We'll be using [degit](https://github.com/Rich-Harris/degit) to clone the example locally, without any of the git history
+
 ```bash
-# create a new project in my-awesome-app
-npx degit xataio/examples/apps/sample-next-auth-app my-awesome-app
+npx degit xataio/examples/apps/sample-next-auth-app your/project/name
 ```
 
 ## Initialize your Database
 
-In order to connect to a workspace, a `XATA_API_TOKEN` and a database URL. One of way of setting it up is running the [Xata CLI](https://xata.io/docs/getting-started/cli).
+First, make sure you've globally installed the [Xata CLI](https://xata.io/docs/getting-started/cli).
 
-> 💡 Having it globally will speed-up linking your project via `xata init`, just make sure you have it up-to-date so it generates the SDK with compatible types.
+```bash
+npm install --location=global @xata.io/cli@latest
+```
 
-Once you've created a project and installed dependencies with `npm install` (or `pnpm install` or `yarn`), you need to initialize your database:
+Once installed we'll authorize our Xata account using the `auth` command:
 
 ```bash
-npm run xata:init
+xata auth login
 ```
 
-> ⚠️ By default this command runs with `npx`, adjust your `package.json` accordingly, specially if you have the CLI globally available in your system.
+Finally we have to do a few things:
 
-### Regenerating Types
+- Create a new database
+- Create the schema to match Auth.js' [expectations](https://authjs.dev/reference/adapter/xata#setup)
+- Connect our new database to our project & generate the Xata client
 
-Once the project is already linked, re-generate your types every time the schema changes via
+Thankfully, the CLI can handle all of this using the `init` command (since we already have a local json schema).
 
-```
-npm run xata:codegen
+```bash
+xata init --schema=schema.template.json --codegen lib/xata.codegen.ts
```

Comment:
I still think there should be a mention on the possibility of running the CLI with `npx` as an alternative just to make it clear the starter _can_ run without any need of external system configuration.

Reply:
I may be wrong, but I assume the user, if already an `npx` user, will have the knowledge that they could use it instead. I'd rather only point the user in a singular, cohesive direction throughout our documentation & starters. 

That fair?

Reply:
You may be right, but assumptions are dangerous for documentation. Specially in the case where `npx <package>` yields potentially a different result than `npx <package>@latest`. I'd prefer things to be explicit considering this bug is a hard to find because it bits silently.

And also, consider that there's no mention to this alternative in the official docs, and that most starters are self-contained. This one not only assumes a system-wide dependency, but it also implies the user is aware it needs to be updated before running the tasks.